src/AuthModule/Components/Register/Register.css

```diff
@@ -0,0 +1,74 @@
+
```

Comment:
These styles will be seen by all pages in the application, I suggest making this CSS  module file `Register.module.css` to make the CSS  specific to the file that you are importing to.

Having a general CSS will cause CSS specificity war especially when the application grows, and you 'll find that the classes here apply to another page in which you have other styles.